opdracht-4.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+  // Opdracht 4
+  const output4 = document.querySelector('.output4');
+  const output4e = document.querySelector('.output4e');
+
+  // Opdracht 4a
+  function tvFormat(arrayIndex) {
+    const brandTv = arrayIndex.brand;
+    const typeTv = arrayIndex.type;
+    const nameTv = arrayIndex.name;
+    return `${brandTv} ${typeTv} - ${nameTv}`;
+  }
+
+  // Opdracht 4b
+  function tvPriceFormat(arrayIndex) {
+    const price = arrayIndex.price;
+    return `€${price}`;
+  }
+
+  // Opdracht 4c
+  function showScreenSizes(arrayIndex) {
+    let outputSizes = '';
+    for (let i = 0; i < arrayIndex.availableSizes.length; i++) {
+      const sizeInch = arrayIndex.availableSizes[i]; // Inches
+      const sizeCm = (arrayIndex.availableSizes[i] * 2.54).toFixed(0); // Centimeters
+
+      if (i === arrayIndex.availableSizes.length - 1) { // Country information part 2 opdracht
+        return outputSizes += `${sizeInch} inch (${sizeCm} cm)`;
+      } else {
+        outputSizes += `${sizeInch} inch (${sizeCm} cm) | `;
+      }
```

Comment:
Om de code wat te ontdubbelen:
```suggestion
      outputSizes += `${sizeInch} inch (${sizeCm} cm)`;
      
      if (i < arrayIndex.availableSizes.length - 1) { // Country information part 2 opdracht
        outputSizes += " | ";
      }
```

Reply:
DRY is altijd goed :-)